src/map/handler/Map.Drag.js

```diff
@@ -96,12 +96,12 @@ export var Drag = Handler.extend({
 	},
 
 	_onDown: function () {
```

Comment:
There is a difference though, previously you could just tap the map during a flyTo, and it would interrupt the animation. With this change, tapping does nothing, you actually have to pull on the map to stop the animation.